src/components/CommentCard/CommentCard.jsx

```diff
@@ -1,16 +1,37 @@
+import { deleteComment } from "../../utils/utils";
 import "./CommentCard.css";
 
-export default function CommentCard({ comment }) {
+export default function CommentCard({ comment, setComments, setDeleteMessage }) {
   const formattedDate = new Date(comment.created_at).toLocaleString();
 
+  function handleDelete() {
+    deleteComment(comment.comment_id)
+      .then(() => {
+        setComments((prevComments) => {
+          return prevComments.filter(
+            (prevComment) => prevComment.comment_id !== comment.comment_id
+          );
+        });
```

Comment:
It might be worth doing this first and to give the user visual representation straight away, then in the catch block you could address that the comment wasn't deleted as you already have.